docs/self-hosted-appcircle/install-server/helm-chart/configuration/production-readiness/s3-compatible-storage-configuration.md

```diff
@@ -0,0 +1,267 @@
+  -n appcircle \
+  --from-literal=accessKey=<YOUR_ACCESS_KEY> \
+  --from-literal=secretKey=<YOUR_SECRET_KEY>
+```
+
+  </TabItem>
+  <TabItem value="openshift" label="OpenShift">
+
+- Create the project that Appcircle Server will be installed in if you haven't yet:
+
+```bash
+oc new-project appcircle
+```
+
+- **Create a OpenShift secret** named `<helm-release-name>-minio-connection` with your S3-compatible access and secret keys:
+
+```bash
+oc create secret generic appcircle-server-minio-connection \
+  -n appcircle \
+  --from-literal=accessKey=<YOUR_ACCESS_KEY> \
+  --from-literal=secretKey=<YOUR_SECRET_KEY>
+```
+
+  </TabItem>
+</Tabs>
+
+:::caution
+- Replace `appcircle` with your actual namespace if different.
+- Replace `<YOUR_ACCESS_KEY>` and `<YOUR_SECRET_KEY>` with your actual access and secret keys.
+- Replace `appcircle-server-minio-connection` with `<helm-release-name>-minio-connection`. Appcircle documentation uses `appcircle-server` as the release name.
+:::
+
+### 5. Configure Appcircle Server to Use S3-Compatible Storage
+
+**Configure your `values.yaml` file** to use your S3-compatible object storage.
+
+Add or update the following configuration to your `values.yaml` file:
+
+<Tabs groupId="s3-provider">
+
+  <TabItem value="minio" label="MinIO">
+
+```yaml
+global:
+  minio:
+    url: "https://your-minio-endpoint.com" # e.g., https://minio.spacetech.com:9000
+    region: "local" # MinIO uses "local" as the region
+    useHttp: "false" # Set to "true" if the MinIO endpoint uses HTTP instead of HTTPS
+    bucketPrefix: "appcircle-spacetech-" # Replace with your actual bucket prefix
+resource:
+  s3:
+    clientProvider: "MINIO" # Use "MINIO" for MinIO
+minio:
+  enabled: false # Disable the internal MinIO server if you are using an external MinIO server
+```
+
+  </TabItem>
+
+  <TabItem value="any-s3-compatible-provider" label="Any S3-Compatible Provider">
+
+```yaml
+global:
+  minio:
+    url: "https://your-s3-endpoint.com" # e.g., https://s3.wasabisys.com
+    region: "us-east-1" # Use the region of the buckets
+    useHttp: "false" # Set to "true" if the S3 endpoint uses HTTP instead of HTTPS
+    bucketPrefix: "appcircle-spacetech-" # Replace with your actual bucket prefix
+resource:
+  s3:
+    clientProvider: "AWS" # Use "AWS" for AWS S3-compatible providers
+minio:
+  enabled: false # Disable the internal MinIO server if you are using an external MinIO server
+```
+
+  </TabItem>
+
+</Tabs>
+
+:::caution
+- Set `useHttp` to `true` only if your S3 or MinIO endpoint does not support HTTPS (not recommended for production).
+- Check your provider's documentation for the correct endpoint URL and region.
+- Replace `appcircle-spacetech-` with your actual bucket prefix.
+:::
+
+## Next Steps
+
+After completing the S3-compatible storage configuration:
+
+1. **Return to the main installation guide**:
+   - [Installation](/self-hosted-appcircle/install-server/helm-chart/installation)
```

Comment:
The 1st step of the "next steps" can be in sync with the same section in GCP or AWS pages.